client/state/products-list/selectors/get-product-price-tiers.ts

```diff
@@ -7,11 +7,26 @@ import 'calypso/state/products-list/init';
 import type { AppState } from 'calypso/types';
 
 type Product = {
-	price_tiers: PriceTiers;
+	price_tier_list: PriceTierEntry[];
 };
 
-export type PriceTiers = Record< string, PriceTier >;
+export interface PriceTierEntry {
+	minimum_units: number;
+	maximum_units?: undefined | null | number;
+	per_unit_fee: number;
+	per_unit_fee_display: string;
+	per_unit_fee_monthly_display: string;
+	flat_fee: number;
+	flat_fee_display: string;
+	flat_fee_monthly_display: string;
+	undiscounted_per_unit_fee?: undefined | null | number;
+	transform_quantity_divide_by?: undefined | null | number;
+	transform_quantity_round?: undefined | null | 'up' | 'down';
+}
 
+/**
+ * @deprecated use PriceTierEntry
+ */
 export type PriceTier =
 	| {
 			flat_price: number;
```

Comment:
I think we can safely delete this type since is no longer used anywhere else.

Reply:
Agreed.